packages/backend/src/index.internal.ts

```diff
@@ -1,4 +1,6 @@
 export * from './index.js';
+// eslint-disable-next-line @typescript-eslint/naming-convention
+import * as __export__function_runtime from './function/runtime/index.js';
```

Comment:
```suggestion
import * as __export__function__runtime from './function/runtime/index.js';
```

`__` is converted to `/` in api checks.